prot.c

```diff
@@ -1,11 +1,29 @@
+#include "prot.h"
```

Comment:
Better after the `<>` includes?

Reply:
The idea is that this enforces prot.h to be self-contained. It will fail to build if it misses a type or something similar. If you put it after the system headers they could provide the missing types and hide the error.

Reply:
Makes sense. Let's try to do this in more places.